src/js/api/api.js

```diff
@@ -842,15 +842,6 @@ export default function Api(element) {
             return core.createInstream();
         },
 
-        /**
-         * Calls `skipAd` on the active instream adapter instance if present.
-         * @returns {Api} The Player API instance.
-         */
-        skipAd() {
-            core.skipAd();
-            return this;
```

Comment:
Should there still be a way for customers to force an ad to be skipped (or end an ad break)? This removes that capability for the vast plugin in some cases.

Reply:
We've been discussing this very thing and the consensus is that we should only allow skipping the ad if:

- there is a `skipoffset` configured
- we have reached the `skipoffset`
- if a VPAID, the VPAID's `getAdSkippableState()` method returns `true`

cc @pajong @vseventer @egreaves - thoughts?